tiles/src/main/java/com/protomaps/basemap/layers/Roads.java

```diff
@@ -52,15 +52,17 @@ public void processFeature(SourceFeature sourceFeature, FeatureCollector feature
         .setAttrWithMinzoom("oneway", sourceFeature.getString("oneway"), 14);
 
       if (highway.equals("motorway") || highway.equals("motorway_link")) {
-        feat.setAttr("pmap:kind", "highway").setZoomRange(6, 15);
+        // TODO: (nvkelso 20230622) Use Natural Earth for low zoom roads at zoom 5 and earlier
+        //       as normally OSM roads would start at 6, but we start at 3 to match Protomaps v2
+        feat.setAttr("pmap:kind", "highway").setZoomRange(3, 15);
 
         if (highway.equals("motorway")) {
           feat.setAttrWithMinzoom("ref", shield_text, 7)
-            .setAttrWithMinzoom("ref_length", shield_text_length, 7)
+            .setAttrWithMinzoom("shield_text_length", shield_text_length, 7)
```

Comment:
Switch to Tilezen forward naming on the new v3 property for `shield_text_length` (instead of `ref_length`).